backend/app/routes/bot.py

```diff
@@ -16,6 +16,7 @@
     BotSwitchVisibilityInput,
     EmbeddingParams,
     Knowledge,
+    GenerationConfig,
```

Comment:
As well as repo/model, could you rename to GenerationConfigParams or GenerationParams?